tests/src/Functional/ContactsDashboardTest.php

```diff
@@ -11,7 +11,7 @@
  */
 class ContactsDashboardTest extends BrowserTestBase {
 
-  public static $modules = ['user', 'contacts', 'views'];
+  public static $modules = ['user', 'views', 'help', 'block'];
 
```

Comment:
This is missing documentation. An `@inheritdoc` with an additional comment as to why we're enabling those modules would be good.

On those requirements, are they hard requirements, or can we put them in as soft dependencies (i.e. wrap their use in checks)?